js/adapt-contrib-xapi.js

```diff
@@ -5,26 +5,23 @@
  *              - Barry McKay <barry@learningpool.com>
  *              - Brian Quinn <brian@learningpool.com>
  */
-define([
-  'core/js/adapt',
-  'core/js/enums/completionStateEnum',
-  'libraries/async.min',
-  'libraries/xapiwrapper.min',
-  'libraries/url-polyfill.min'
-], function(Adapt, COMPLETION_STATE, Async) {
+import Adapt from 'core/js/adapt';
+import COMPLETION_STATE from 'core/js/enums/completionStateEnum';
+import Async from 'libraries/async.min';
+import XAPIWrapper from 'libraries/xapiwrapper.min';
+import urlPolyfill from 'libraries/url-polyfill.min';
```

Comment:
I think the framework now has a `URL` polyfill in it already.
Worth removing this and checking `window.URL` can still be found in IE11 where it's currently natively unavailable.

Reply:
The file referenced from the libraries folder can be deleted.